cmd/manager/main.go

```diff
@@ -2,11 +2,10 @@ package main
 
 import (
 	"fmt"
-	"go.uber.org/zap"
-	"os"
-
 	"github.com/mongodb/mongodb-kubernetes-operator/pkg/apis"
 	"github.com/mongodb/mongodb-kubernetes-operator/pkg/controller"
+	"go.uber.org/zap"
```

Comment:
we need to have a gofmt and goimports commit hook, so that these don't move around at each PR